salon_creator/lib/app.dart

```diff
@@ -15,42 +15,21 @@ class SalonCreatorApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      theme: ThemeData(
-        fontFamily: 'NotoSansJP',
-        primaryColor: primaryColor,
-        textButtonTheme: TextButtonThemeData(
-          style: ButtonStyle(
-            foregroundColor: MaterialStateProperty.all(Colors.white),
-          ),
-        ),
-        dividerColor: Color.fromRGBO(193, 193, 193, 1),
-        appBarTheme: AppBarTheme(
-          shadowColor: Colors.transparent,
-          color: Colors.white,
-          foregroundColor: Colors.black,
-          textTheme: TextTheme(
-            headline6: TextStyle(
-              color: Colors.black,
-              fontSize: 20,
-              fontFamily: 'NotoSansJP',
-            ),
-          ),
-        ),
-      ),
+      theme: _buildThemeData(),
       routes: {
         '/salon_creation': (context) => SalonCreationScreen(),
         '/home': (context) => HomePage(),
-        '/login': (context) => LoginPage(),
+        '/login': (context) => LoginScreen(),
         '/salon_registration': (context) => SalonRegistrationScreen(),
-        '/screen': (context) => RegistrationSuccessScreen(),
+        '/RegistrationSuccess': (context) => RegistrationSuccessScreen(),
```

Comment:
```suggestion
        '/registration_success': (context) => RegistrationSuccessScreen(),
```
This also needs to be in snake case